Makefile.am

```diff
@@ -452,11 +454,34 @@ libwx_new_la_LIBADD = $(WX_LIBS)
 
 # auxiliary executables
 
+# The 'md5sum' binary is redistributed to msw end users for
```

Comment:
Not sure why do we need this comment here, it seems completely unrelated to `md5sum_XXX` definitions. Unless I'm missing something, I'd just remove it.